.github/workflows/sec-workflow.yml

```
@@ -67,7 +71,10 @@ jobs:
           build-scan-terms-of-use-agree: "yes"
 
   sonarcloud-analysis:
-    if: ${{ inputs.with-sonarcloud }}
+    # Also skip when FIXERS_SONAR_TOKEN is unavailable (e.g. Dependabot PRs,
+    # fork PRs) — GitHub withholds repo secrets from those workflow runs, so
+    # the Sonar scan/quality-gate steps below would just fail auth.
+    if: ${{ inputs.with-sonarcloud && secrets.FIXERS_SONAR_TOKEN != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)